src/main/java/com/spotify/logging/LoggingConfigurator.java

```diff
@@ -433,6 +427,15 @@ public static void configure(final File file, final String defaultIdent) {
     StatusPrinter.printInCaseOfErrorsOrWarnings(context);
   }
 
+  private static LoggerContext setupLoggerContext(Logger rootLogger, String ident) {
+    final LoggerContext context = rootLogger.getLoggerContext();
+    context.reset();
+    context.putProperty("ident", ident);
+    context.putProperty("pid", getMyPid());
+    context.putProperty("hostname", getHeliosHostname());
```

Comment:
when this method returns null, is something inside the syslog appender defaulting to the default hostname?